pkg/pkgmgr/rpm.go

```diff
@@ -646,8 +646,11 @@ func (rm *rpmManager) unpackAndMergeUpdates(ctx context.Context, updates unversi
 
 								done <<< "$(echo "$json_str" | tr -d '{}\n' | tr ',' '\n')"
 
+								# Convert OS_VERSION from X.Y.Z to X.Y format
+                                OS_VERSION_XY=$(echo "$OS_VERSION" | cut -d'.' -f1-2)
```

Comment:
looks like there are spaces here while others are aligned with tabs, can we make this consistent